Source/BidMachineAdapterBannerAd.swift

```diff
@@ -39,11 +39,21 @@ final class BidMachineAdapterBannerAd: BidMachineAdapterAd, PartnerAd {
 
         self.loadCompletion = completion
 
-        // There's no harm in setting the placement ID when loading a bidding ad, but calling
-        // .withPayload(request.adm ?? "") causes an error when BidMachine parses the empty string
-        config.populate { $0.withPlacementId(request.partnerPlacement) }
         if let adm = request.adm {
             config.populate { $0.withPayload(adm) }
+        } else {
+            config.populate {
+                let setting = self.request.partnerSettings["price"]
+                let priceNSNum = setting as? NSNumber
+                guard let price = priceNSNum?.doubleValue else {
+                    let error = error(.loadFailureInvalidAdRequest)
+                    self.log(.loadFailed(error))
+                    completion(.failure(error))
+                    return
+                }
```

Comment:
Can we refactor this code into a helper function somewhere to avoid repeating in all ad files?

Reply:
Maybe in BicMachineAdapterAd, something like a 
```
func populateConfig(_ config: BidMachineRequestConfigurationProtocol) -> Error? {
 ...
}
```

Reply:
[Computed property added as discussed on Slack.](https://github.com/ChartBoost/chartboost-mediation-ios-adapter-bidmachine/pull/6/commits/9a2b83fe8872fea6faf74f14cd738ed46102efb1)

Reply:
ty!